Packs/Base/Scripts/StixParser/StixParser.py

```diff
@@ -1253,31 +1268,26 @@ def decode(props, **kwargs):
 
         hashes = props.find_all('Hash')
         for h in hashes:
-            htype = h.find('Type')
-            if htype is None:
-                continue
-            htype = htype.string.lower()
-            if htype not in ['md5', 'sha1', 'sha256', 'ssdeep']:
-                continue
-
             value = h.find('Simple_Hash_Value')
             if value is None:
                 continue
             value = value.string.lower()
             value_list = value.split('##comma##')
             for v in value_list:
-                result.append({
-                    'indicator': v.strip(),
-                    'htype': htype,
-                    'type': 'File'
-                })
+                v = v.strip()
+                if type := detect_indicator_type_file(v):
+                    result.append({
+                        'indicator': v.strip(),
```

Comment:
```suggestion
                        'indicator': v,
```

looks like v is already striped.

Reply:
nice!